### DISCUSSION TITLE
Is it possible to download only DLC but not the base game?

### DISCUSSION THREAD

**Discussion by Project-Magenta** [legendary-gl/legendary#321] [Q&A]
I want to download only a DLC for game without downloading the games

**Comment by nutterthanos** [comment]
Why would you want to imo?

**Reply by Project-Magenta** [reply]
Because I don't have the disk space for the full game and I am only interested in a specific file (I won't tell you the answer because you won't help me)

**Reply by nutterthanos** [reply]
Well I have already tried you can't
Edit: you can just add --no-install to the install command
